frontend/src/app/shared/directives/autofocus/autofocus.directive.ts

```diff
@@ -4,10 +4,10 @@ import {
 import { FocusHelperService } from 'core-app/shared/directives/focus/focus-helper';
 
 @Directive({
-  selector: '[autoFocus]',
+  selector: '[opAutoFocus]',
```

Comment:
What about merging this with the `focus` directive? However, I am also fine with doing it separately.